src/configs/template_test_config.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
This file should be named as `template_evaluation_config.json`

Reply:
Fixed